doc/source/multivisit.rst

```diff
@@ -3,4 +3,435 @@
+
+The MVM filename defined for the input exposures follows the convention.
+
+  `hst_skycell-p<PPPP>x<XX>y<YY>_<instr>_<detector>_<filter>_<ipppssoo>_fl[ct].fits`
+
+where:
+  ==========  =========================================================================
+  Element     Definition
+  ==========  =========================================================================
+  <PPPP>      ProjectionCell ID as a zero-padded 4 digit integer
+  <XX>,<YY>   SkyCell ID within ProjectionCell as zero-padded 2 digit integers
+  <instr>     Name of HST instrument from **INSTRUME** header keyword
+  <detector>  Name of detector from **DETECTOR** header keyword
+  <filter>    Name of filter from **FILTER** or **FILTER1,FILTER2** header keyword(s)
+  <ipppssoo>  Pipeline-assigned **IPPPSSOO** designation from original input filename
+  fl[ct]      Suffix of either **flt** or **flc** from original input filename
+  ==========  =========================================================================
+
+This insures that each exposure gets renamed in a way that allows them to be easily identified with respect to the
+output SkyCell layer the exposure contributes to during MVM processing.
+
+
+Generate Input File
+--------------------
+The MVM processing could simply combine whatever input files are present in the current working directory.  However,
+that may result in working with more than 1 SkyCell at a time which can, for some steps, end up requiring more memory
+or disk space than is available on the system.  Therefore, the code relies on an input `poller file` which specifies exactly
+what files should be processed at one time.  This input `poller file` will typically only contain the names of exposures which
+overlap only a single SkyCell regardless of instrument, detector or any other observational configuration.  This file
+is simply an ASCII file with one of 2 formats.
+
+If all the exposures which are intended to be processed are already in the current working directory, then this file can
+be a simple list of filenames, perhaps even just a subset of what is in the current directory.  This form allows anyone
+to decide what they want to combine together without exposures that are unrelated to their research.  This file gets
+interpreted and converted into the second more informative form by opening each of the input files and extracting the
+necessary supplemental information from header keywords.
+
+Generating one of these basic input files with the filename *skycell_basic_input.out* can be done using commands such as:
+
+.. code:: python
+
+    import glob
+
+    flcs = sorted(glob.glob("hst*fl?.fits"))
+    with open("skycell_basic_input.out", "w") as fout:
+        _ = [fout.write(f"{fname}\n") for fname in flcs]
+
+This file can then be used as input to the top-level MVM processing code using:
+
+.. code:: python
+
+    from drizzlepac import hapmultisequencer
+    rv = hapmultisequencer.run_mvm_processing(input_filename)
```

Comment:
For this example, change "input_filename" to "skycell_basic_input.out".

Reply:
Changed in 241a672